### PR TITLE
Validate tokens

### PR DESCRIPTION
**Pull request by cizmazia**

### CODE REVIEW

File: service/signIdentityCertificate.go [hunk 1]
```diff
@@ -9,12 +9,10 @@ import (
 )
 
 func (r *RequestHandler) SignIdentityCertificate(ctx context.Context, req *pb.SignCertificateRequest) (*pb.SignCertificateResponse, error) {
-	if req.AuthorizationContext == nil {
-		return nil, logAndReturnError(status.Errorf(codes.InvalidArgument, "cannot sign identity certificate: invalid AuthorizationContext"))
+	if err := r.auth(req.GetAuthorizationContext().GetAccessToken()); err != nil {
```
Comment (jkralik) [inline service/signIdentityCertificate.go]: please leave req.AuthorizationContext -> if it is nil it cause segfault.

File: service/signCertificate.go [hunk 2]
```diff
@@ -9,15 +9,13 @@ import (
 )
 
 func (r *RequestHandler) SignCertificate(ctx context.Context, req *pb.SignCertificateRequest) (*pb.SignCertificateResponse, error) {
-	if req.AuthorizationContext == nil {
-		return nil, logAndReturnError(status.Errorf(codes.InvalidArgument, "cannot sign identity certificate: invalid AuthorizationContext"))
+	if err := r.auth(req.GetAuthorizationContext().GetAccessToken()); err != nil {
```
Comment (jkralik) [inline service/signCertificate.go]: please leave req.AuthorizationContext -> if it is nil it cause segfault.

Comment (cizmazia) [reply service/signCertificate.go]: the getters `GetAuthorizationContext().GetAccessToken()` handle this

File: service/signIdentityCertificate.go (hunk 1 above)
Comment (cizmazia) [reply service/signIdentityCertificate.go]: the getters `GetAuthorizationContext().GetAccessToken()` handle this

**Pull request closed without merging**